addons/test_website_slides_full/__manifest__.py

```diff
@@ -13,7 +13,6 @@
         'website_sale_slides',
```

Comment:
Commit message:

As the purpose of this commit is performance, it's important to state how long it's taking after the changes (on average).
(You can also specify the percentage of improvement, always nice to see).

Reply:
@awa-odoo Done. :smile: